bin/ext4mag

```diff
@@ -0,0 +1 @@
+echo "extract all;ext2sim labels on;ext2sim -R -C;ext2spice scale off;ext2spice -F -R -C -f spice3;calma" | magic -dnull -noconsole $1
```

Comment:
I wonder if it would work to simply have this as executable with a magic shebang?

```
#!/usr/bin/env magic -dnull -noconsole
extract all
ext2sim labels on
```

or a shell script with a herestring

```
magic -dnull -noconsole << EOF
EOF
```

Reply:
deleted